s4e-backend/src/test/java/pl/cyfronet/s4e/controller/InstitutionControllerTest.java

```diff
@@ -0,0 +1,172 @@
+
+    @BeforeEach
+    public void beforeEach() {
+        testDbHelper.clean();
+        appUser = appUserRepository.save(AppUser.builder()
+                .email(PROFILE_EMAIL)
+                .name("Get")
+                .surname("Profile")
+                .password(passwordEncoder.encode("password"))
+                .enabled(true)
+                .build());
+
+        slugInstitution = slugService.slugify(testInstitution);
+        Institution institution = institutionRepository.save(Institution.builder()
+                .name(testInstitution)
+                .slug(slugInstitution)
+                .build());
+        Group group = groupRepository.save(Group.builder().name("__default__").slug("default").institution(institution).build());
+
+        UserRole userRole = UserRole.builder().role(AppRole.GROUP_MEMBER).user(appUser).group(group).build();
+        userRoleRepository.save(userRole);
+    }
+
+    @AfterEach
+    public void afterEach() {
+        testDbHelper.clean();
+    }
+
+    @Test
+    public void shouldGetUserInstitution() throws Exception {
+        mockMvc.perform(get(API_PREFIX_V1 + "/institutions")
+                .contentType(MediaType.APPLICATION_JSON)
+                .with(jwtBearerToken(appUser, objectMapper)))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$[0].name").value(is(equalTo(testInstitution))))
+                .andExpect(jsonPath("$[0].slug").value(is(equalTo(slugInstitution))));
+    }
+
+    @Test
+    public void shouldGetEmptySet() throws Exception {
+        AppUser appUser = appUserRepository.save(AppUser.builder()
+                .email("get2@profile.com")
+                .name("Get")
+                .surname("Profile")
+                .password(passwordEncoder.encode("password"))
+                .enabled(true)
+                .build());
+
+        mockMvc.perform(get(API_PREFIX_V1 + "/institutions")
+                .contentType(MediaType.APPLICATION_JSON)
+                .with(jwtBearerToken(appUser, objectMapper)))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.length()", is(equalTo(0))));
+    }
+
+    @Test
+    public void shouldGetAllUsersInstitutions() throws Exception {
+        String testInstitution = "Test Institution ZK";
+        String slugInstitution = slugService.slugify(testInstitution);
+        Institution institution = institutionRepository.save(Institution.builder()
+                .name(testInstitution)
+                .slug(slugInstitution)
+                .build());
+        Group group = groupRepository.save(Group.builder().name("__default__").slug("default").institution(institution).build());
+
+        UserRole userRole = UserRole.builder().role(AppRole.GROUP_MEMBER).user(appUser).group(group).build();
+        userRoleRepository.save(userRole);
+
+        testInstitution = "Test Institution PAK";
+        slugInstitution = slugService.slugify(testInstitution);
```

Comment:
And not redefine them here as well.
Use names such as `testInstitutionZK`, `slugInstZK`, etc.